CONTRIBUTING.md

```diff
@@ -0,0 +1,249 @@
+ 
+Please make sure you have used DVC correctly to rerun building the model. 
+
+#### Tool-kit
+Coming soon
+
+#### General
+
+Before a PR can be merged, it needs to be approved by one core developer. 
+Please prefix the title of your pull request with [MRG] if the contribution is 
+complete and should be subjected to a detailed review. An incomplete 
+contribution – where you expect to do more work before receiving a full 
+review – should be prefixed [WIP] (to indicate a work in progress) and 
+changed to [MRG] when it matures. For a more detailed explanation of what 
+these mean to the organisation please see [Project 
+Management](#Project-Management)
+
+In order to ease the reviewing process, we recommend that your contribution 
+complies with the following rules before marking a PR as [MRG]. The bolded 
+ones are especially important:
+
+- Give your pull request a helpful title that summarises what your 
+contribution does. This title will often become the commit message once merged 
+so it should summarise your contribution. In some cases “Fix <ISSUE 
+TITLE>” is enough. “Fix #<ISSUE NUMBER>” is never a good title.
+- Make sure your code passes the tests.
+- Make sure your code is properly commented and documented, and make sure the 
+documentation renders properly
+- Tests are necessary for enhancements to be accepted. 
+- Make sure that your PR does not add any linter violations.
+
+
+Follow the Coding guidelines.
+
+    When applicable, use the validation tools and scripts.
+
+    Often pull requests resolve one or more other issues (or pull requests). 
+If merging your pull request means that some other issues/PRs should be 
+closed, you should use keywords to create link to them (e.g., Fixes #1234; 
+multiple issues/PRs are allowed as long as each one is preceded by a keyword). 
+Upon merging, those issues/PRs will automatically be closed by GitHub. If your 
+pull request is simply related to some other issues/PRs, create a link to them 
+without using the keywords (e.g., See also #1234).
+
+    New features often need to be illustrated with narrative documentation in 
+the user guide, with small code snipets. If relevant, please also add 
+references in the literature, with PDF links when possible.
+
+
+### CI/CD
+
+We have Circle CI in two of our code repos.
+
+### Branch Naming Convention
+We use a branch for each major release, and tags on that branch for each
+ minor release.
+
+For example, beta users will use a beta or a release candidate, which will 
+have a 0.2beta1 release name and tag on the 0.2.X branch. 
+
+
+## Contributing Anything Other than Code
+Non code contributions are vital to this project. A prime example of this is
+ sharing papers and notes. We do this via the wiki attached to this project.
+
+
+## Code of Conduct
+
+> Please note that this project is released with a [Contributor Code of 
+Conduct](https://github.com/malteserteresa/opt-out/blob/master/CODE_OF_CONDUCT.
```

Comment:
I believe this link is wrong? it points to Teresa's repo and not to the OOT one. Not 100% sure though.